deps/verifier/src/tdx/eventlog.rs

```diff
@@ -124,6 +124,7 @@ impl CcEventLog {
 
 /// Defined in TCG PC Client Platform Firmware Profile Specification section
 /// 'UEFI_PLATFORM_FIRMWARE_BLOB Structure Definition'
+#[allow(dead_code)]
```

Comment:
I assume this is unrelated?

Reply:
It was to remove a warning when compiling, as the TDX library is unused.